### PR TITLE
[SEMVER-MAJOR] Switch to oracledb

### PR DESCRIPTION
**Pull request by raymondfeng**
Connect to strongloop/loopback-connector-oracle#62

### CODE REVIEW

**Comment by bajtos** [comment]
@rmg @sam-github would you mind taking a look and helping us with reviewing this change? It's about installers.

File: bin/installers/Linux/installer.sh [hunk 1]
```diff
@@ -6,13 +6,13 @@ SCRIPT_DIR=$(dirname "${BASH_SOURCE[0]}")
 INSTALL_DIR=$(cd -P -- "$SCRIPT_DIR/.." && pwd -P)
 source "$INSTALL_DIR/../../lib/utils"
 
-setup_configuration "$OCI_DIR" "$HOME/strong-oracle.rc"
+setup_configuration "$OCI_DIR" "$HOME/oracledb.rc"
 cat << MYEOF
 -------------------------------------------------------------------------------
-"strong-oracle.rc" has been created in \$HOME. Please manually add the
+"oracledb.rc" has been created in \$HOME. Please manually add the
```
Comment (sam-github) [inline bin/installers/Linux/installer.sh]: ~/.bash_profile is only run for login shells, isn't it? I always start my shells as login shells, but last time I saw, that wasn't true of new terminals with iTerm or most multi-tab terminal emulators. Or perhaps I misremember.

**Comment by sam-github** [comment]
If its semver major, it should bump the semver in the package.json, or it will do bad things when auto-published to CI.

**Comment by rmg** [comment]
@raymondfeng Sorry, you'll have to rebase again. I just updated the license and copyright info on master.

**Comment by raymondfeng** [comment]
@sam-github I bumped the major version.
@rmg Rebased.

File: lib/download.js [hunk 2]
```diff
@@ -86,13 +89,28 @@ module.exports = function (url, version, dest, cb) {
...
-        cb && cb();
-      }).on('error', function (error) {
-        cb && cb(error);
-      });
+        var err;
+        var home = homedir();
+        var icDir = path.join(home, 'oracle-instant-client');
+        if (!fs.existsSync(icDir)) {
+          try {
+            fse.copySync(path.join(dest, 'instantclient'), icDir,
```
Comment (rmg) [inline lib/download.js]: Is there a reason not to do this check before downloading and just extracting the tar stream directly into the destination if it doesn't already exist?

Comment (rmg) [inline lib/download.js]: I'm also a little worried about how this sync call may interfere with error propagation in the surrounding async code. Might not be a real problem, but I at least find it hard to reason about.

Comment (raymondfeng) [inline lib/download.js]: We bundle Oracle instant client with the pre-built `oracledb` addon. Upon installation, the installer tries to download it from one tarball.

Comment (rmg) [inline lib/download.js]: It looks to me like:
```
stream tar from (disk or http)
extract tar to dst
if icDir not exists:
  copy dst to icDir
```
I'm suggesting:
```
if icDir not exist:
  stream tar from (disk or http)
  extract tar to icDir
```

Ignore me if I'm misreading it, though, of it there's some other reason multiple copies are needed.

Comment (raymondfeng) [inline lib/download.js]: The tarball contains two parts:
- ic
- oracledb.node

**Pull request merged**